homeassistant/components/lutron/binary_sensor.py

```diff
@@ -42,20 +50,20 @@ class LutronOccupancySensor(LutronDevice, BinarySensorEntity):
     _attr_device_class = BinarySensorDeviceClass.OCCUPANCY
 
     @property
-    def is_on(self):
+    def is_on(self) -> bool:
```

Comment:
By the way, do you mind adding these types in a preliminary PR? It's not a deal-breaker, but it would make this PR a little more scoped :)

Reply:
I have done some config flows in the past and I think I should have enough knowledge to help you make this across the finish line :)